foundation/foundation-archetype/src/main/resources/archetype-resources/__rootArtifactId__-deploy/src/main/resources/templates/remote-auth-secret.yaml

```diff
@@ -0,0 +1,10 @@
+# Convert this to a SealedSecret using kubeseal and replace it here
```

Comment:
I: Moved this to the archetype so that projects wouldn't have to deal with it constantly regenerating if they a) didn't need it or b) renamed it to something more explicit to their project.